ProjectSettings/InputManager.asset

```diff
@@ -59,8 +59,8 @@ InputManager:
     descriptiveNegativeName: 
     negativeButton: down
     positiveButton: up
-    altNegativeButton: s
-    altPositiveButton: w
+    altNegativeButton: w
+    altPositiveButton: s
```

Comment:
I think this is inverted now. s should be negativeButton and w should be positiveButton.